browser/src/App.tsx

```diff
@@ -56,7 +56,7 @@ function App() {
         <div className="fadeOutOnScroll">
           <Hero />
         </div>
-        <div style={{ backdropFilter: `blur(3px)` }}>
+        <div>
```

Comment:
is this removing the glassmorphism?

Reply:
just for the essay section!

Reply:
ah gotcha i think we'll have problems with the blobs then but @alejandrogarciasalas if you want to look into something more performant that'd be great!

Reply:
yeah! I have a maxed out m1 mbp so things feel ok perf wise, will do some testing with throttling 👍